build-system/tasks/e2e/describes-e2e.js

```diff
@@ -130,7 +130,7 @@ async function createPuppeteer(opt_config = {}) {
  * @param {string} browserName
  * @param {!SeleniumConfigDef=} args
  * @param {?string} deviceName
- * @return {!SeleniumDriver}
+ * @return {!WebDriver}
  */
 function createSelenium(browserName, args = {}, deviceName) {
```

Comment:
For accuracy, should this function be renamed to `createWebDriver`?

Reply:
`createSelenium` keeps consistent with the `EngineType` enum. It's misleading because `SeleniumDriver` isn't a thing :see_no_evil: